SETUP.md

```diff
@@ -66,6 +66,18 @@ To perform backup, BBQ needs rights to read BigQuery data from the project which
     ```
 * (Optionally) Configure schedule time and kinds to export in [cron.yaml](./config/cron.yaml) file.
 
+### Security setup
```

Comment:
I would move this setup above to the installation steps as it looks like additional config.

Here in Security setup you can document security layers, i.e. firewall, IAP, GAE admin endpoints.

Reply:
👍